src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private StringBuilder stringBuilder;
+    private String resultString;
```

Comment:
```suggestion
```
these variables should not be class variables. if you want to pass result of first method to second, then first method should return this result instead of assigning it to class variable